### PR TITLE
ניסוי 2

### PR DESCRIPTION
**Pull request by tomhess**

### CODE REVIEW

File: helloworld.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-print("hellooo Ron1111")
+print("hellooo Ron11222333 ve ze ba bash1")
```
Comment (selotape) [inline helloworld.py]: 1. No more hebrew please (it hurts my eyes)
2. In Python, prefer single quotes (`'my single quote string'`) over double quotes (`"this is less pythonic"`)

Fix and then we'll merge your code into `master`

**Review by selotape — CHANGES_REQUESTED** [review]
see comment in code

**Comment by tomhess** [comment]
is it okey now ?

**Review by selotape — APPROVED** [review]
Lol, looks better now.

**Pull request merged**